tests/foreman/ui/test_reporttemplates.py

```diff
@@ -414,9 +415,11 @@ def test_positive_schedule_generation_and_get_mail(
                 'email_to': 'root@localhost',
             },
         )
-    file_path = PurePath('/tmp/').joinpath(f'{gen_string("alpha")}.json')
+    randstring = f'{gen_string("alpha")}'
+    file_path = PurePath('/tmp/').joinpath(f'{randstring}.json')
```

Comment:
You can choose to use pythons tmp library here instead of joinpath with /tmp.

Just an improvement non blocking comment.